commands/command-template/example-command.js

```diff
@@ -0,0 +1,84 @@
+// Imports the discord.js-commando library which is required for commands to work. 
+const { Command } = require('discord.js-commando');
+
+// Create the embedded message layout for the bots message. 
+// Im still not 100% sure what value the embed color value is from but if/
+// this website is very helpful with generating the needed color and code
+// https://leovoel.github.io/embed-visualizer/ 
```

Comment:
```suggestion

/* Define the contents and structure of embedded message for bot response.
Colour Codes for embed messages are in decimal format, see bellow website for help:
https://leovoel.github.io/embed-visualizer/ 
https://convertingcolors.com/ */
```

Comments should be clear, factual and informative. Please ensure you don't put unclear questioning statements in them. If you are unsure of something, ask, or research or leave it out.